resources/assets/js/components/StrategicTalentResponse/ResponseScreening/ApplicantBucket.tsx

```diff
@@ -171,6 +176,159 @@ const AlertWhenUnsaved = (): React.ReactElement => {
   return <></>;
 };
 
+interface ReferenceEmailModalProps {
+  id: string;
+  parent: Element | null;
+  email: Email | null;
+  visible: boolean;
+  onConfirm: () => void;
+  onCancel: () => void;
+}
+
+const ReferenceEmailModal: React.FC<ReferenceEmailModalProps> = ({
+  id,
+  parent,
+  email,
+  visible,
+  onConfirm,
+  onCancel,
+}): React.ReactElement => {
+  const renderAddress = (adr: EmailAddress): string =>
+    `${adr.name} <${adr.address}>`.trim();
```

Comment:
Doesn't look like the `trim()` here does much, unless the `adr.name` has leading spaces.

Reply:
Good point